### PR TITLE
 Added files for zulu translations using JW300 corpus

### PR DESCRIPTION
**Pull request by Ari-Ramkilowan**

### CODE REVIEW

File: en-zu/jw300-baseline/config.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    max_sent_length: 100
+    src_vocab: "/content/drive/My Drive/masakhane/en-zu-baseline/vocab.txt"
+    trg_vocab: "/content/drive/My Drive/masakhane/en-zu-baseline/vocab.txt"
+
+testing:
+    beam_size: 5
+    alpha: 1.0
+
+training:
+    load_model: "/content/drive/My Drive/masakhane/en-zu-baseline/pretrained/enzu4000baseline/101000.ckpt" # if uncommented, load a pre-trained model from this checkpoint
```
Comment (jaderabbit) [inline en-zu/jw300-baseline/config.yaml]: can we make these relative? 
@juliakreutzer What's your preference here?

Comment (juliakreutzer) [reply en-zu/jw300-baseline/config.yaml]: I think it's fine :) these models will have to be downloaded from individual links anyway... So at the moment one has to adapt the `load_model` and vocabulary paths manually in order to use pretrained models.

**Comment by juliakreutzer** [comment]
@jaderabbit your requested changes should be resolved now as well.

**Comment by juliakreutzer** [comment]
@Ari-Ramkilowan Can you please adapt the location to the new "benchmarks" directory?
@jaderabbit I think we should be ready to approve the PR, could you please approve the changes? (Otherwise we can't merge) 
Thanks!

**Comment by juliakreutzer** [comment]
Ok, moved the directory myself. @jaderabbit we need your final approval to merge ;)

**Pull request merged**